engine/runtime/src/main/java/org/enso/interpreter/runtime/number/EnsoBigInteger.java

```diff
@@ -16,7 +16,7 @@
 /** Internal wrapper for a {@link BigInteger}. */
 @ExportLibrary(InteropLibrary.class)
 @ExportLibrary(TypesLibrary.class)
-public final class EnsoBigInteger implements TruffleObject {
+public final class EnsoBigInteger extends Number implements TruffleObject {
```

Comment:
Got a comment from Christian Humer:

> Your trick with `EnsoBigInteger extends Number` will not work on Espresso for example.

I guess we'll have to switch to official `BigInteger` interop [once it is available](https://github.com/oracle/graal/pull/5490)